### ISSUE TITLE
Increased circulating type 1 collagen N-terminal telopeptide level

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11247]
**Preferred term label:**
Increased circulating type 1 collagen N-terminal telopeptide level

**Synonyms**

**Definition (free text, please give PubMed ID)**
https://uts.nlm.nih.gov/uts/umls/concept/C5780203

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
https://www.ncbi.nlm.nih.gov/books/NBK616087/

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001412
- label: Elevated circulating C-terminal telopeptide cross-link of type1 collagen concentration
- parents: [Abnormal circulating collagen degradation product concentration (HP:0033386)](https://hpo.jax.org/app/browse/term/HP:0033386)
- pmid: PMID:40638774
- def: The concentration of C-terminal telopeptide cross-link of type1 collagen in the blood circulation is above the upper limit of normal.
- comment: C-terminal telopeptide cross-link of type1 collagen is a biomarker of bone resorption.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11247
- synonyms:

**Comment by MickeySegal** [comment]
Shouldn't this be N terminal as in https://www.ncbi.nlm.nih.gov/books/NBK616087/ ?  

Also, we had the urinary counterpart https://hpo.jax.org/browse/term/HP:0032208.  Should we harmonize the nomenclature to something like:  N-terminal telopeptide cross-link of type1 collagen

I don't know if the serum and urine elevations run in parallel and which is used.  We have one disease listed for serum and 5 listed for urine, with no overlap.  It would be good to know if both serum and urine elevations apply to all 6 diseases.
